vj4/model/builtin.py

```diff
@@ -71,6 +71,10 @@
 PERM_EDIT_TRAINING = 1 << 48
 PERM_EDIT_TRAINING_SELF = 1 << 49
 
+# Record.
+PERM_EDIT_RECORD_VISIBILITY = 1 << 50
+PERM_EDIT_RECORD_VISIBILITY_SELF = 1 << 51
```

Comment:
@iceb0y Shall we store permission in binary format in mongodb? We could serialize a long int to binary via something like `xx.to_bytes(16, byteorder='little')`